src/conn/conn_layered.c

```diff
@@ -214,6 +225,9 @@ __disagg_pick_up_checkpoint(WT_SESSION_IMPL *session, uint64_t meta_lsn, uint64_
      */
     WT_RELEASE_WRITE(conn->disaggregated_storage.global_checkpoint_id, checkpoint_id + 1);
 
+    /* Update the checkpoint timestamp. */
+    WT_RELEASE_WRITE(conn->disaggregated_storage.last_checkpoint_timestamp, checkpoint_timestamp);
+
```

Comment:
This is not a problem for this ticket, but we should keep in mind that these two (checkpoint_id and timestamp) are not changed together atomically.